package.json

```diff
@@ -1,52 +1,70 @@
+  "version": "0.1.1",
   "description": "Generic Replicache backend on Next.js",
   "homepage": "https://github.com/rocicorp/replicache-nextjs",
   "repository": "github:rocicorp/replicache-nextjs",
-  "main": "out/index.js",
   "license": "Apache-2.0",
   "scripts": {
     "format": "prettier --write 'src/*.{js,jsx,json,ts,tsx,html,css,md}' '*.{cjs,js,jsx,json,ts,tsx,html,css,md}'",
     "check-format": "prettier --check 'src/*.{js,jsx,json,ts,tsx,html,css,md}' '*.{cjs,js,jsx,json,ts,tsx,html,css,md}'",
     "lint": "eslint --ext .ts,.tsx,.js,.jsx src/",
-    "build-cjs": "tsc --module commonjs --outDir out",
-    "build-esm": "tsc --module ESNext --outDir out && mv out/index.js out/index.mjs",
-    "build": "npm run build-esm && npm run build-cjs",
-    "prepack": "npm run lint && npm run test && npm run build",
-    "test": "env TS_NODE_COMPILER_OPTIONS='{\"module\": \"commonjs\" }' mocha -r ts-node/register 'src/**/*.test.ts'"
+    "build": "rm -rf out && tsc",
+    "prepack": "npm run lint && npm run build",
+    "pretest": "npm run build",
+    "test": "mocha --ui=tdd 'out/**/*.test.js'"
+  },
+
+  "peerDependencies": {
+    "react": ">=16.0 <18.0",
+    "next": ">=12.1.6",
+    "react-dom": ">=16.0 <18.0",
+    "replicache": ">=11.0.0",
+    "pg-mem": ">=2.5.0"
   },
   "devDependencies": {
-    "@rocicorp/logger": "^2.2.0",
-    "@rocicorp/rails": "^0.2.1",
-    "@supabase/supabase-js": "^1.35.3",
     "@types/chai": "^4.3.0",
     "@types/mocha": "^9.1.0",
-    "@types/node": "^14.14.37",
-    "@types/pg": "^8.6.4",
-    "@types/react": "^17.0.11",
-    "@types/react-dom": "^18.0.5",
     "@typescript-eslint/eslint-plugin": "^5.3.1",
     "@typescript-eslint/parser": "^5.18.0",
     "chai": "^4.3.6",
-    "classnames": "^2.3.1",
-    "eslint": "^8.2.0",
     "mocha": "^9.2.1",
-    "nanoid": "^3.3.1",
-    "next": "11.0.1",
-    "pg": "^8.7.3",
-    "pg-mem": "^2.5.0",
     "prettier": "^2.2.1",
-    "react": "^17.0.2",
-    "react-dom": ">=16.0 <18.0",
-    "replicache": "^11.0.0",
+    "nanoid": "^3.3.1",
     "ts-node": "^10.7.0",
     "typescript": "^4.1.5",
-    "zod": "^3.17.3"
+    "zod": "^3.17.3",
+    "classnames": "^2.3.1",
+    "eslint": "^8.2.0",
+    "@types/node": "^14.14.37",
+    "@types/pg": "^8.6.4",
+    "@types/react": "^17.0.11",
+    "@types/react-dom": "^18.0.5",
+    "pg-mem": "^2.5.0",
+    "pg": "^8.7.3",
+    "@supabase/supabase-js": "^1.35.3",
+    "@rocicorp/logger": "^2.2.0",
+    "@rocicorp/rails": "^0.2.1",
+    "next": "^12.1.6",
+    "react": "17.0.2"
   },
+  "type": "module",
   "files": [
-    "out/index.js",
-    "out/index.mjs",
-    "out/index.d.ts"
-  ]
+    "out/*",
+    "!out/*.test.*"
+  ],
+  "exports": {
+    "./out/backend": {
+      "types": "./out/backend/index.d.ts",
+      "import": "./out/backend/index.js"
+    },
+    "./out/frontend": {
+      "types": "./out/frontend/index.d.ts",
+      "import": "./out/frontend/index.js"
+    },
+    "./out/backend/supabase": {
```

Comment:
I was having trouble  getting next to ignore supabase backend dependencies ignored for the frontend. However, I think this can be undone now now that everything is ESM